Formula/c/codeberg-cli.rb

```diff
@@ -0,0 +1,24 @@
+class CodebergCli < Formula
+  desc "CLI for Codeberg"
+  homepage "https://codeberg.org/Aviac/codeberg-cli"
+  url "https://codeberg.org/Aviac/codeberg-cli/archive/v0.4.7.tar.gz"
+  sha256 "a8d1356faab84076f14977652dabbfcad4411f49beb4d11a1bc0ee8936bd1d6c"
+  head "https://codeberg.org/Aviac/codeberg-cli.git", branch: "main"
```

Comment:
```suggestion
  license "AGPL-3.0-only"
  head "https://codeberg.org/Aviac/codeberg-cli.git", branch: "main"
```

Reply:
it is actually AGPL-3.0-or-later, https://codeberg.org/Aviac/codeberg-cli/src/branch/main/Cargo.toml#L5 :)